### DISCUSSION TITLE
Output label from pipeline diarization

### DISCUSSION THREAD

**Discussion by Gabrielk99** [pyannote/pyannote-audio#808] [General]
Hi, thanks a lot for the framework and for the patience to answer all questions that i asked here.

Well, i was like know if i can do the label output of diarization pipeline  be equal my labels from my rttm files. If answer will yes, how?

**Comment by Yagna24** [comment]
Hi @Gabrielk99 , Can you guide me where the rttm files are located?

**Reply by Gabrielk99** [reply]
Which rttm? The output from diarization pipeline? If yes, The default is the same Path that your pipeline config file is.

**Reply by Yagna24** [reply]
Hi @Gabrielk99 , I cant seem to find the config file, where is it located generally? can you please share the path where its located in your case ?

**Reply by Gabrielk99** [reply]
Well, if you are using pyannote 1.1.0, and execute the pipeline diarization by tutorial, the path is tutorials/pipelines/speaker_diarization/train/Protocol.SpeakerDiarization.ProtocolDataName.development/apply/"some code"/Protocol.SpeakerDiarization.ProtocolDataName.test.rttm

**Reply by Yagna24** [reply]
Thank you for the answer
